smoke-test/src/main/resources/opennms-overlay/etc/org.opennms.features.datachoices.cfg

```diff
@@ -1,3 +0,0 @@
-#Mon May 27 04:17:39 UTC 2019
-enabled=false
```

Comment:
This was here so that the feature would be disabled by default in smoke test runs. Is this still the case?

Reply:
I think the deletion was an accident. I reverted.